vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -0,0 +1,57 @@
+# How to customize VizroAI
+
+This guide explains how to customize `VizroAI`.
+
+`VizroAI` accepts a single argument named `model`. This `model` argument can either be a string that specifies the name of the `ChatOpenAI` model or an instantiated `ChatOpenAI` model.
+
+We'll delve into customizing the `ChatOpenAI` model later; for now, let's concentrate on the default usage of `VizroAI`.
+
+## Default usage
+
+`VizroAI`can be initialized without any arguments, in which case it will, by default, use OpenAI model `"gpt-3.5-turbo"` with a temperature setting of 0.
+`"gpt-3.5-turbo"` points to the latest model offering enhanced speed and accuracy in generating responses in requested formats, while maintaining cost-effective performance.
+
+
+## Choosing OpenAI model
+
+When choosing OpenAI model, there are few things to keep in mind; the complexity of the task, cost associated with it and the desired response time.
+
+**gpt-3.5** model series are good in both general tasks and chat-specific applications. These models have lower price point and higher speeds for providing answers.
+
+However, for more demanding tasks, consider upgrading to the **gpt-4** models. While they are part of a more capable GPT model series, their response time is slower than gpt-3.5 models, and they come at a higher cost.
+
+In the example below, we customized `VizroAI` by providing OpenAI model name in a string form. ( see for [models currently supported by Vizro-AI](../explanation/faq.md#which-llms-are-supported-by-vizro-ai)).
+
+!!! example "Customize with string"
+
+    === "python"
+        ```py linenums="1"
+        from vizro_ai import VizroAI
+
+        vizro_ai = VizroAI(model="gpt-3.5-turbo-0125")
+        ```
+
+## Customize ChatOpenAI instance
```

Comment:
I would add somewhere in this section on where to import from, and what we support again, with a hint that we are working on more vendors.